autoware_perception_msgs/msg/TrafficSignal.msg

```diff
@@ -0,0 +1,2 @@
+int64 traffic_signal_id
+autoware_perception_msgs/TrafficLightElement[] elements
```

Comment:
What difference is between TrafficSignal.msg and TrafficLight.msg?

Reply:
TrafficSignal: Traffic signal as concept. This is one for each direction of the intersection.
TrafficLight: Traffic signals as equipments. This can be multiple for each direction of the intersection.

Reply:
I see. LGTM